examples/websearch/websearch-tools.ts

```diff
@@ -71,10 +64,10 @@ async function main() {
   console.log('----- Prompt:', messages.find((m) => m.role === 'user')?.content, '\n')
```

Comment:
Can we also clean these up? Just have an ordering of prompt first, model output, function call + result and then the model output after that